src/app/embed/adapters/adapter.properties.ts

```diff
@@ -2,6 +2,7 @@ import { Observable } from 'rxjs/Observable';
 
 export const PropNames = [
   'audioUrl',
+  'duration',
```

Comment:
Seems like a mistake that this wasn't here already.  (Which caused the MergeAdapter to not pick up `duration`).